src/Middleware/CorsMiddleware.php

```diff
@@ -46,11 +47,18 @@ public function process(ServerRequestInterface $request, RequestHandlerInterface
             throw InvalidConfigurationException::fromInvalidPipelineConfiguration();
         }
 
-        if (! $this->cors->isCorsRequest($request)) {
+        try {
+            $isCorsRequest = $this->cors->isCorsRequest($request);
+        } catch (InvalidOriginValueException $exception) {
```

Comment:
It is an exception right now.
To avoid having a bc break, I think its feasible, no?

Besides that: 99,9% of requests are valid, we experienced issues with maybe 1/1.000.000 requests in our application. If that is not an exception, I wonder what is an exception then?

Reply:
Fine by me 👍